src/DnsResolver/docs/New-AzDnsResolverDnsForwardingRuleset.md

```diff
@@ -0,0 +1,274 @@
+---
+external help file:
+Module Name: Az.DnsResolver
+online version: https://docs.microsoft.com/powershell/module/az.dnsresolver/new-azdnsresolverdnsforwardingruleset
+schema: 2.0.0
+---
+
+# New-AzDnsResolverDnsForwardingRuleset
+
+## SYNOPSIS
+Creates or updates a DNS forwarding ruleset.
+
+## SYNTAX
+
+```
+New-AzDnsResolverDnsForwardingRuleset -Name <String> -ResourceGroupName <String> -Location <String>
+ [-SubscriptionId <String>] [-IfMatch <String>] [-IfNoneMatch <String>]
+ [-DnsResolverOutboundEndpoint <ISubResource[]>] [-Tag <Hashtable>] [-DefaultProfile <PSObject>] [-AsJob]
+ [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates or updates a DNS forwarding ruleset.
+
+## EXAMPLES
+
+### Example 1: Create a DNS forwarding ruleset 
+```powershell
+PS C:\> New-AzDnsResolverDnsForwardingRuleset -Name dnsForwardingRuleset -ResourceGroupName sampleRG -Location westus2 -DnsResolverOutboundEndpoint  @{id = "/subscriptions/ea40042d-63d8-4d02-9261-fb31450e6c64/resourceGroups/sampleRG/providers/Microsoft.Network/dnsResolvers/sampleResolver/outboundEndpoints/sampleOutbound";}
```

Comment:
Can we demonstrate using the outbound endpoint we just created to create a dns forwarding ruleset? For example
```powershell
$outboundEndpoint = new-azdnsresolveroutboundendpoint ******
new-azdnsresolverdnsforwardingruleset -DnsResolverOutboundEndpoint  $outboundEndpoint 
```